packages/expo-cellular/src/Cellular.ts

```diff
@@ -1,12 +1,16 @@
 import { UnavailabilityError } from '@unimodules/core';
+import { Platform } from '@unimodules/react-native-adapter';
+import { PermissionResponse, PermissionStatus } from 'expo-modules-core';
 
-import { CellularGeneration } from './Cellular.types';
+import { CellularGeneration, CellularInfo } from './Cellular.types';
 import ExpoCellular from './ExpoCellular';
 
 export { CellularGeneration };
 
 // @needsAudit
 /**
+ * @deprecated field, use `getCurrentCellularInfoAsync()` instead
```

Comment:
Can you add the internal link to the recommended method? (probably `#getcurrentcellularinfoasync`)